content/docs/command-reference/params/index.md

```diff
@@ -43,7 +43,9 @@ to define [stage](/doc/command-reference/run) dependencies more granularly:
 changes to other parts of the params file will not affect the stage. Parameter
 dependencies also prevent situations where several stages share a regular
 dependency (e.g. a config file), and any change in it invalidates all of them
-(see `dvc status`), causing unnecessary re-executions upon `dvc repro`.
+(see `dvc status`), causing unnecessary re-executions upon `dvc repro`. However,
+you can track all the parameters in a file by setting _parameter value_ to be
+`null`.
```

Comment:
Yeah, this is tricky. What about this?

```suggestion
you can track all the parameters in a file by specifying the value of
`params.yaml:` to be `null` or empty.
```

Reply:
I modified the sample instead (similar to the `dvc.yaml` ref.) and generalized the text in the description. See 0ef8c607